daemon/logger/logentries/logentries.go

```diff
@@ -51,7 +51,10 @@ func New(info logger.Info) (logger.Logger, error) {
 	}
 	var lineOnly bool
 	if lineOnly, err = strconv.ParseBool(info.Config[lineonly]); err != nil {
-		return nil, errors.Wrap(err, "error parsing lineonly option")
```

Comment:
Instead of this, would it work to just check if the `info.Config[lineonly]` is empty before trying to parse it?